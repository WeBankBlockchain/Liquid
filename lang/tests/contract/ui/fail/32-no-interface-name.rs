use liquid_lang as liquid;

#[liquid::interface]
mod foo {
    extern "liquid" {
        fn bar();
    }
}

fn main() {}