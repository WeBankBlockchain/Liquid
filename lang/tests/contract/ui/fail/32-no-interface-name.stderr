error: expected `name` parameter in `#[liquid::interface]` attribute
 --> $DIR/32-no-interface-name.rs:3:1
  |
3 | #[liquid::interface]
  | ^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `liquid::interface` (in Nightly builds, run with -Z macro-backtrace for more info)
